src/ui/server/constants/field-ids/insurance/your-buyer/index.ts

```diff
@@ -7,6 +7,9 @@ const YOUR_BUYER = {
     WEBSITE: 'website',
     FIRST_NAME: 'firstName',
     LAST_NAME: 'lastName',
+    POSITION: 'position',
+    EMAIL: 'email',
+    CAN_CONTACT_BUYER: 'canContactBuyer',
```

Comment:
Not a blocker but when we get to saving the data might want to tweak the namings